lib/matplotlib/image.py

```diff
@@ -293,6 +293,15 @@ def get_size(self):
 
         return self._A.shape[:2]
 
+    def get_shape(self):
+        """Return the shape of the image as
+        tuple (numrows, numcols, channels).
+        """
```

Comment:
```suggestion
        """
        Return the shape of the image as tuple (numrows, numcols, channels).
        """
```

Reply:
Can you use the numpydoc format for the returns?

Reply:
I was going to do that but line length exceeded pep8 limit.